packages/core/package.json

```diff
@@ -51,7 +51,7 @@
     "http-proxy-agent": "^5.0.0",
     "https-proxy-agent": "^5.0.0",
     "iconv-lite": "^0.6.0",
-    "inversify": "^6.0.1",
```

Comment:
Inversify `6.1.3` was just released, containing the build fix. So we should already indicate that we want to use that.

Reply:
Yeah, good idea.